### ISSUE TITLE
Use readonly reference to identifiers instead of creating a computed value

### ISSUE THREAD

**Issue by tscpp** [debrixjs/debrix#21]
#### Summary
Currently, all binding values will be wrapped in `$computed` (until #20 is fixed). This is so the dependencies of the computed can be observed by the binder.

Except in some cases where static values are used instead of reactive properties (e.g 1+1). If no there exist no identifiers in the binding value, then the expression should not be wrapped in a `$computed`.

The same goes for when only a identifier is specified (the whole expression is one identifier). Except this expression needs to be observed. However, because the expression is an identifier, the computed will always return the same exact value as the identifier. There is no point in computing it again. In this case, the identifier should be readonly referenced (`$ref` but read-only).

#### TL;DR
Static expressions (e.g. 1+1) should not be wrapped with `$computed`. When the whole expression is one identifier, then the property should be referenced, except read-only (`$ref` but read-only).

#### Examples
```js
(1 + 1) -> (1 + 1)
(name) -> ($readRef(name))
(firstName + lastName) -> ($computed(() => firstName + lastName))
```
